### PR TITLE
search beers filter

### PR DESCRIPTION
**Pull request by dwantula**

### CODE REVIEW

File: src/screens/SearchBeers/SearchBeers.jsx [hunk 1]
```diff
@@ -19,6 +19,8 @@ const alcohollVolumeOptions = [
   },
 ]
 
+
+
```
Comment (mi-chal) [inline src/screens/SearchBeers/SearchBeers.jsx]: No need to add extra lines

File: src/screens/SearchBeers/SearchBeers.jsx [hunk 2]
```diff
@@ -101,14 +104,16 @@ class SearchBeersComponent extends PureComponent {
     this.setState({
       [name]: value
     });
+    console.log(value)
```
Comment (mi-chal) [inline src/screens/SearchBeers/SearchBeers.jsx]: Unnecessary console.log

File: src/screens/SearchBeers/SearchBeers.jsx [hunk 3]
```diff
@@ -93,6 +147,10 @@ class SearchBeersComponent extends PureComponent {
       colorOfBeer: '',
       brewedBefore: '',
       food: '',
+      beerName: [],
```
Comment (mi-chal) [inline src/screens/SearchBeers/SearchBeers.jsx]: beerName should not be an array and it is not - it's a string

File: src/screens/SearchBeers/SearchBeers.jsx [hunk 4]
```diff
@@ -171,12 +235,53 @@ class SearchBeersComponent extends PureComponent {
...
+          <Heading
+            text="Search beer"
+            type="h4"
+          />
+          <InputComponent
+            name="beerName"
+            onChange={this.handleChange}
+            value={beerName}
+            placeholder="Write name of beer"
+            option='beer_name='
```
Comment (mi-chal) [inline src/screens/SearchBeers/SearchBeers.jsx]: Input doesn't have option

File: src/screens/SearchBeers/SearchBeers.jsx [hunk 5]
```diff
@@ -83,6 +84,59 @@ const foodOptions = [
...
+]
+
+const hopsOptions = [
+  {
+    label: 'Fuggles',
+    value: 'hops=Fuggles'
+  },
+  {
+    label: 'First Gold',
+    value: 'hops=First Gold'
```
Comment (mi-chal) [inline src/screens/SearchBeers/SearchBeers.jsx]: Does api handle spaces? I don't think so  ("if you need to add spaces just add an underscore (_)" - it is from the documentation). Also you should not use capital letters - it should be like `first_gold`

**Review by mi-chal — APPROVED** [review]

File: src/screens/SearchBeers/SearchBeers.jsx (hunk 1 above)
Comment (dwantula) [reply src/screens/SearchBeers/SearchBeers.jsx]: Done

File: src/screens/SearchBeers/SearchBeers.jsx (hunk 2 above)
Comment (dwantula) [reply src/screens/SearchBeers/SearchBeers.jsx]: Done!

File: src/screens/SearchBeers/SearchBeers.jsx (hunk 3 above)
Comment (dwantula) [reply src/screens/SearchBeers/SearchBeers.jsx]: Done!

File: src/screens/SearchBeers/SearchBeers.jsx (hunk 4 above)
Comment (dwantula) [reply src/screens/SearchBeers/SearchBeers.jsx]: Done!

File: src/screens/SearchBeers/SearchBeers.jsx (hunk 5 above)
Comment (dwantula) [reply src/screens/SearchBeers/SearchBeers.jsx]: Done!

**Pull request merged**